mmv1/products/monitoring/MonitoredProject.yaml

```diff
@@ -17,7 +17,7 @@ base_url: v1/locations/global/metricsScopes
 create_url: v1/locations/global/metricsScopes/{{metrics_scope}}/projects
 delete_url: v1/locations/global/metricsScopes/{{metrics_scope}}/projects/{{name}}
 self_link: v1/locations/global/metricsScopes/{{metrics_scope}}
-id_format: v1/locations/global/metricsScopes/{{metrics_scope}}/projects/{{name}}
+id_format: locations/global/metricsScopes/{{metrics_scope}}/projects/{{name}}
```

Comment:
I think we need an upgrade state function to make sure any of these resources that were created with the `v1/` prefix are moved to the correct id format